generator/final_processor_country.cpp

```diff
@@ -310,6 +310,7 @@ void CountryFinalProcessor::AddFakeNodes()
 void CountryFinalProcessor::DropProhibitedSpeedCameras()
 {
   static auto const speedCameraType = classif().GetTypeByPath({"highway", "speed_camera"});
+  ASSERT_NOT_EQUAL(speedCameraType, Classificator::INVALID_TYPE, (speedCameraType));
```

Comment:
Why it can be invalid? What was the warning?

Reply:
Strange, because it is used below. Does the warning disappear if you move this static variable into the lambda, right where it is used?

Reply:
> because it is used below.

Only conditionally and two scopes deeper.

> Does the warning disappear if you move this static variable into the lambda, right where it is used

Yes, and I think this is the best solution. So I have changed it.